apps/ejabberd/rebar.config

```diff
@@ -16,11 +16,6 @@
                deprecated_function_calls,
                deprecated_functions]}.
 
-{ct_extra_params, "-pa test "
-                  "-ct_hooks ct_progress_hook "
```

Comment:
Nothing seems to use `apps/ejabberd/src/ct_progress_hook.erl` now; should we remove it? There's still a copy at `test.disabled/ejabberd_tests/src/ct_progress_hook.erl` for big tests.